core/modules/taxonomy/taxonomy.admin.inc

```diff
@@ -906,8 +928,9 @@ function taxonomy_form_term($form, &$form_state, TaxonomyTerm $term = NULL, Taxo
   else {
     $path = '<front>';
   }
-  $options = backdrop_parse_url($path);
-  $options['attributes']['class'][] = 'form-cancel';
+
+  $cancel_options = backdrop_parse_url($path);
+  $cancel_options['attributes']['class'][] = 'form-cancel';
```

Comment:
Why not set this directly in `#options` the `$form['actions']['cancel']` link? That would make it consistent with https://github.com/backdrop/backdrop/pull/3669/files#diff-f504d9eb38a71e38ce014b49cf9e126f2c3eee6677ca8261425b67e7bf7856b4R232